pkg/config/setup/config.go

```diff
@@ -997,8 +997,9 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("otelcollector.extension_timeout", 0)         // in seconds, 0 for default value
 	config.BindEnvAndSetDefault("otelcollector.submit_dummy_metadata", false) // dev flag - to be removed
 	config.BindEnvAndSetDefault("otelcollector.converter.enabled", true)
+	config.BindEnvAndSetDefault("otelcollector.gateway.enabled", false)
```

Comment:
@songy23 I need to be able to read whether gateway mode is enabled in order to have the converter choose between ddflare vs oss datadog extension. I added this config option, let me know if this is how you envisioned it ?